samples/corda/corda-simple-application/clients/src/main/resources/config/credentials/Corda_Network/verification-policy.json

```diff
@@ -6,5 +6,19 @@
       "type": "Signature",
       "criteria": ["PartyA"]
     }
+  },
+  {
+    "pattern": "localhost:10006#net.corda.samples.tokenizedhouse.flows.GetAssetPledgeStatusByPledgeId:*",
+    "policy": {
+      "type": "Signature",
+      "criteria": ["PartyA"]
+    }
+  },
+  {
+    "pattern": "localhost:10006#com.cordaSimpleApplication.flow.GetAssetPledgeStatusByPledgeId:*",
+    "policy": {
+      "type": "Signature",
+      "criteria": ["PartyA"]
+    }
   }]
-}
\ No newline at end of file
+}
```

Comment:
Don't we need a policy for the claim query too?

Reply:
Done.